### PR TITLE
RTVI 0.2.0

### PR DESCRIPTION
**Pull request by marcus-daily**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# 0.2.0
```
Comment (marcus-daily) [inline CHANGELOG.md:1]: There's probably some stuff missing from this changelog, I'll go through one more time and check but if you notice anything please let me know!

Comment (marcus-daily) [reply CHANGELOG.md:1]: Also, I still need to add the version number in the initial request.

Comment (filipi87) [reply CHANGELOG.md:1]: Nice, we were really missing a changelog file. I will add one on iOS as well.

File: rtvi-client-android/src/main/java/ai/rtvi/client/Legacy.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package ai.rtvi.client
+
+
+@Deprecated("VoiceClient is renamed to RTVIClient")
```
Comment (filipi87) [inline rtvi-client-android/src/main/java/ai/rtvi/client/Legacy.kt:4]: Cool, I think I had added a comment somewhere else if we should add these alias. 
It makes sense to keep in its own file.

File: rtvi-client-android/src/main/java/ai/rtvi/client/RTVIClientOptions.kt [hunk 3]
```diff
@@ -5,9 +5,15 @@ import ai.rtvi.client.types.ServiceRegistration
 import ai.rtvi.client.types.Value
 
 /**
- * Configuration options when instantiating a [VoiceClient].
+ * Configuration options when instantiating a [RTVIClient].
  */
-data class VoiceClientOptions(
+data class RTVIClientOptions(
```
[the remark this answers is not in the document]
Comment (filipi87) [reply rtvi-client-android/src/main/java/ai/rtvi/client/RTVIClientOptions.kt:10]: I saw that we are already doing this in a different file. Cool!

**Review by filipi87 — APPROVED** [review]
Nice job!

**Pull request merged**